src/util/evented.js

```diff
@@ -89,7 +89,7 @@ export class Evented {
      * @param {Function} listener The function to be called when the event is fired the first time.
      * @returns {Object} `this`
      */
-    once(type: string, listener: Listener) {
+    once(type: *, listener: Listener) {
```

Comment:
I think this should remain `string`. Any reason this had to change?

Reply:
@kkaefer flow fails without `*` and `on` currently uses `*` as well (assuming for similar reasons).

Reply:
Ah, flow seems to have trouble matching a string to a string literal :/